supertokens_python/recipe/multifactorauth/multi_factor_auth_claim.py

```diff
@@ -0,0 +1,245 @@
+from __future__ import annotations
+
+from typing import Any, Dict, Optional, Set
+
+from supertokens_python.recipe.session.interfaces import (
+    ClaimValidationResult,
+    JSONObject,
+    SessionClaim,
+    SessionClaimValidator,
+)
+
+from .types import (
+    FactorIdsAndType,
+    MFAClaimValue,
+    MFARequirementList,
+    SessionRecipeUserIdInput,
+)
+from .utils import update_and_get_mfa_related_info_in_session
+
+
+class HasCompletedRequirementListSCV(SessionClaimValidator):
+    def __init__(
+        self,
+        id_: str,
+        claim: MultiFactorAuthClaimClass,
+        mfa_claim_validators: MultiFactorAuthClaimValidators,
```

Comment:
if this is not used, you can remove mfa_claim_validators

Reply:
Removed